xbmc/utils/EmbeddedArt.cpp

```diff
@@ -21,15 +21,17 @@
 #include "EmbeddedArt.h"
 #include "Archive.h"
 
-EmbeddedArtInfo::EmbeddedArtInfo(size_t siz, const std::string &mim)
+EmbeddedArtInfo::EmbeddedArtInfo(size_t siz,
+                                 const std::string &mim, const std::string& t)
```

Comment:
please name it `type` instead of `t` like you did further down

Reply:
then i would have to rename the member first.

Reply:
succinct variable names are OK if they're documented in the header